charts/consul/values.yaml

```diff
@@ -85,7 +85,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: "hashicorp/consul:1.10.3"
+  image: "hashicorp/consul:1.10.4"
```

Comment:
oops! I think I forgot to update consul version in chart.yaml

Reply:
i assumed we would do that when we do the next release release 😅